### ISSUE TITLE
Scapulothoracic incongruity

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11070]
**Preferred term label:**
Scapulothoracic incongruity

**Synonyms**

**Definition (free text, please give PubMed ID)**
X-ray or CT scan of the scapula showing incongruity of the scapulothoracic space

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Scapulothoracic crepitus

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Added as HP:0020319
A lack of smooth, physiological alignment or movement between the scapula (shoulder blade) and the posterior thoracic wall (the rib cage). The scapulothoracic joint is unique, as it lacks true synovial articulation and is dynamically controlled through surrounding muscular contractions. The scapula glides on the posterior thorax covered with muscle layers rather than a cartilaginous surface. This feature can be ascertained by radiography or CT scan of the scapula showing incongruity of the scapulothoracic space.
